google-cloud-translate/CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Release History
 
+### 2.0.0 / 2019-10-28
+
+#### ⚠ BREAKING CHANGES
+
+* **translate:** Add Translate V3 client
+
+#### Features
+
+* Update Ruby dependency to minimum of 2.4 ([#4206](https://www.github.com/googleapis/google-cloud-ruby/issues/4206))
+* Add Translate V3 client
+  * Update google-cloud-translate to contain a generated v3 client
+  * The v3 client includes several new features and updates:
+    * Glossaries - Create a custom dictionary to correctly and 
+    * Batch requests - Make an asynchronous request to translate
+    * AutoML models - Cloud Translation adds support for translating
+    * Labels - The Cloud Translation API supports adding user-defined
+  * The following methods now return an instance of
```

Comment:
"now return an instance of" I think the v3 client type needs to be added here.

Reply:
It is, on the next line.

Reply:
Your comment may have been made before I corrected the changelog. The release-please tool does not like commit messages with multi-line list items, and so they have to be fixed manually.